examples_tests/60.ClusteredRendering/clipmap/first_cull.comp

```diff
@@ -0,0 +1,110 @@
+#version 450 core
```

Comment:
put in `nbl/builtin/glsl/ext/clusteredlighting/` and add appropriate `#ifdef/#define` guards and `nbl_glsl_ext_ClusteredLighting_` prefixes to declarations and variable names